### ISSUE TITLE
split altered response to myocardial infarction 

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#3328]
Consider splitting altered response to myocardial infarction MP:0000343 into increased and decreased susceptibility terms.
Change primary term label to myocardial ischemia injury?

This is related to COVID-19

**Comment by sbello** [comment]
Reference for infarction vs ischemia https://academic.oup.com/cardiovascres/article/51/1/13/267395

**Comment by sbello** [comment]
On review it seems that the existing term is talking about the response to infarction not the response to ischemia. However some of the child terms are talking about the response to ischemic injury. We should create a new term for abnormal/altered response to myocardial ischemic injury add increased and decreased suscept. children and move the existing children MP:0003037 and MP:0003038 under appropriate new parent.

**Comment by anna-anagnostop** [comment]
Add new terms:

altered response to myocardial ischemic injury
DEF: altered response to lack of adequate blood flow, due to trauma or disease, to support the normal functioning of the myocardium
SYN: altered response to myocardial ischaemic injury 
SYN: abnormal response to myocardial ischemic injury
SYN: abnormal response to myocardial ischaemic injury

decreased susceptibility to myocardial ischemic injury
SYN: decreased frequency or less severe response, such as necrotic tissue size, to lack of adequate blood flow, due to trauma or disease, to support the normal functioning of the myocardium
SYN: decreased susceptibility to myocardial ischaemic injury
SYN: reduced susceptibility to myocardial ischemic injury
SYN: reduced susceptibility to myocardial ischaemic injury
SYN: increased resistance to myocardial ischemic injury
SYN: increased resistance to myocardial ischaemic injury

increased susceptibility to myocardial ischemic injury
DEF: increased frequency or more severe response, such as necrotic tissue size, to lack of adequate blood flow, due to trauma or disease, to support the normal functioning of the myocardium
SYN: increased susceptibility to myocardial ischaemic injury
SYN: decreased resistance to myocardial ischemic injury
SYN: decreased resistance to myocardial ischaemic injury
SYN: reduced resistance to myocardial ischemic injury
SYN: reduced resistance to myocardial ischaemic injury

**Comment by anna-anagnostop** [comment]
Issues/discrepancies:

1. altered response to myocardial infarction MP:0000343

- textual def uses _cardiac tissue_, not _myocardium_: "change in the physiological response to necrosis of the cardiac tissue, often resulting from the sudden insufficiency of arterial or venous blood supply due to emboli, thrombi or mechanical factors"

(these structures are not identical, see comments in [MP:0010630](http://www.informatics.jax.org/vocab/mp_ontology/MP:0010630)) 

- Current EQ for altered response to myocardial infarction MP:0000343 uses:

'has part' some 
    ('response to'
     and (towards some infarct)
     and ('has modifier' some abnormal))

This EQ lacks the location component (myocardium). Need a better EQ

**Comment by sbello** [comment]
double check annotations to make sure these are specific to myocardial infarction rather than cardiac tissue.
We may need a more general term.

**Comment by anna-anagnostop** [comment]
Changed term label for MP:0000343 to "abnormal response to cardiac infarction", using "altered response to cardiac infarction" as a synonym and removed the EQ [Note: will discuss potential EQ/request new pattern at next Upheno meeting].

Amended definitions for CNS ischemic injury child terms MP:0006059 and MP:0003076 and retinal ischemic injury child terms MP:0012050 and MP:0012049 as proposed above.

Added new terms:
abnormal response to myocardial ischemic injury MP:0031215
decreased susceptibility to myocardial ischemic injury MP:0031216
increased susceptibility to myocardial ischemic injury MP:0031217

Moved the existing MP:0003037 and MP:0003038 under appropriate new parent terms and removed their EQs as they were incorrect.

**Comment by anna-anagnostop** [comment]
‘has_part’ some (‘response to’ and (‘inheres_in’ some ‘cardiac muscle tissue’) and (towards some infarct) and (‘has_modifier’ some abnormal))
Comment: using infarct MPATH:124 is suboptimal, mouse (def: An area of necrosis resulting from a sudden cessation of arterial or venous blood supply)

NOT GOOD
'has part' some ('response to’ and ('inheres in' some (‘necrotic cell death’ and ('part of' some ‘cardiac muscle tissue))) and ('has modifier' some abnormal))
'has_part' some ('response to’ and ('inheres_in' some (%s and ('occurs_in' some %s))) and ('has_modifier' some 'abnormal'))

‘has_part’ some (‘response to’ and (‘inheres_in’ some ‘cardiac muscle tissue’) and towards some ('increased amount' and ('inheres in part of' some ('necrotic cell death')) and (‘has_modifier’ some abnormal))
'has part' some 'response to’ and towards some ('increased amount' and ('inheres in' some 
('necrotic cell death' and ('occurs in' some ‘cardiac muscle tissue’))) and ('has modifier' some abnormal))
